pavelib/assets.py

```diff
@@ -21,13 +22,19 @@
 COFFEE_DIRS = ['lms', 'cms', 'common']
 # A list of directories.  Each will be paired with a sibling /css directory.
 SASS_DIRS = [
-    path("lms/static/sass"),
+#    path("lms/static/sass"),
     path("lms/static/themed_sass"),
     path("cms/static/sass"),
     path("common/static/sass"),
     path("lms/static/certificates/sass"),
 ]
-SASS_LOAD_PATHS = ['common/static', 'common/static/sass']
+SASS_FILES_AND_TARGETS = [
```

Comment:
Since we need to move to declaring all of our app's Sass to CSS compile here, is this complete? Looking at how we bundle assets (https://github.com/edx/edx-platform/blob/master/lms%2Fenvs%2Fcommon.py#L1441), and the ``SASS_DIRS`` up above, I think we still need the Open version of certificates styling in here (unless that is handled based on settings).